session.go

```diff
@@ -2369,6 +2369,11 @@ func (c *Collection) NewIter(session *Session, firstBatch []bson.Raw, cursorId i
 		timeout: -1,
 		err:     err,
 	}
+
+	if socket.ServerInfo().MaxWireVersion >= 4 && c.FullName != "admin.$cmd" {
```

Comment:
Wisdom and I just discussed this.  And realized why the check is there.  It's in case someone was querying system collections, which the find command doesn't support.  So not sure if this check is necessary for getMore, but it doesn't do any harm either.